.chloggen/785.yaml

```diff
@@ -0,0 +1,4 @@
+change_type: enhancement
+component: dotnet
```

Comment:
shouldn't it be `dotnet, dns`?

Reply:
I believe we don't support multiple components in the changelog, so I changed it to `dns`, nothing has changed from the .NET perspective here anyway